src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({ entries, onLike, localSender, fontColor }) => {
+  const message = entries.map((chat) => {
+    return (
+      <ChatEntry
+        key={chat.id}
+        id={chat.id}
+        sender={chat.sender}
+        body={chat.body}
+        timeStamp={chat.timeStamp}
+        liked={chat.liked}
+        onLike={onLike}
+        localSender={localSender}
+        fontColor={fontColor}
+      ></ChatEntry>
```

Comment:
Since the names of the keys on `entry` are the same as the names your are setting on `ChatEntry` attributes/you are using all of the values in `entry` you could do something something like this to save you a few keystrokes: 

```javascript
  const chatComponents = entries.map((entry) => {
    return(
      <ChatEntry
        {...entry}
        onLikeToggle={onLikeBtnToggle}
        key={entry.id}
      />
    );
  });
```

Just recognize that it is not as explicit as what you have. Something to consider!